Open Neuromorphic Research

neural-loop opened the pull request.

neural-loop committed:
feat: Add Open Neuromorphic Research (ONR) program pages

neural-loop committed:
feat: Add program lead card to research section

This commit adds a program lead card to the research section sidebar, displaying the contact person for the ONR program. It includes a new partial "onr/program-lead.html" and updates the research index page and sidebar.

neural-loop committed:
feat(onr): Display program lead card only on research list page

neural-loop committed:
refactor: Improve ONR research pages and guides

This commit refactors the ONR research pages, including:
- Updates to badge system, reviewer, submitter, and criteria guides.
- Improvements to list and single page layouts.
- Adjustments to the page header.

neural-loop committed:
feat: homepage

Add research link to homepage description.

neural-loop committed:
refactor: Move ONR guide pages to /guide subdirectory

The ONR guide pages (submitter, reviewer, criteria, badge) were moved to a `/guide` subdirectory under `/research`.
This commit updates all relevant links and references.

neural-loop committed:
feat: Add research guide documentation

neural-loop committed:
feat: Add research paper registry

This commit introduces the research paper registry with example papers. It includes layouts, content, and an archetype for research papers.

neural-loop committed:
Refactor: Update research page links

Updates the links to the submitter/reviewer guides and research registry to use RelPermalink when the page exists.

neural-loop committed:
docs(research): Update title in research index page

neural-loop committed:
fix(page-header): Use RelPermalink directly

The submitter and reviewer page links now use RelPermalink directly, simplifying the logic.

neural-loop committed:
feat: research: Add ONR program announcement and updates

This commit introduces the ONR program announcement blog post and integrates ONR updates into the research page.
It also adds how-to participate links.

neural-loop committed:
Merge branch 'main' into research

# Conflicts:
#	layouts/index.html

neural-loop committed:
Merge branch 'main' into research

neural-loop committed:
refactor: Refactor homepage layout

Refactors the index.html layout into partials for better organization.

neural-loop committed:
bug: re-add town hall promo

neural-loop committed:
feat(research-papers): Add ONR badge to resource details

neural-loop committed:
Merge branch 'main' into research

neural-loop committed:
docs: Update submitter guide for ONR program

Updated submission guidelines, process, and requirements.
Added OpenReview submission portal link and details.
Clarified acceptable submission formats and prerequisites.

neural-loop committed:
feat: Update ONR program announcement and badge system guide

neural-loop committed:
feat: blog: Update ONR program announcement

neural-loop committed:
chore: Update ONR program announcement date

neural-loop committed:
fix(blog): Update ONR program announcement date and links

neural-loop committed:
Merge branch 'main' into research-tmp

neural-loop committed:
fix(blog): Update ONR announcement banner image

DRosen766 reviewed (commented):
Looks great!  Couple of quick questions

neural-loop commented on layouts/research/list.html:
```diff
@@ -0,0 +1,129 @@
+
+        {{/* --- Guide Links Section --- */}}
+        <div class="mt-16">
+          <h2 class="text-3xl font-bold mb-8 text-center">How to Participate</h2>
```
There are the buttons at the top, and also on the right hand sidebar. Wanted to give the more dynamic updates some place that they will be visible without having to scroll down the page too much.

neural-loop commented on content/neuromorphic-computing/research/guide/submitter-guide/index.md:
```diff
@@ -0,0 +1,112 @@
+
+## Questions?
+
+Join the discussion in the `#onr-program` channel on our [Discord Server](https://discord.gg/hUygPUdD8E) or email us at [contact@open-neuromorphic.org](mailto:contact@open-neuromorphic.org).
```
What do you think would work best? I think #onr-program could work well for a place to direct folks (lets say also if we add volunteer roles around ONR in the furture)

DRosen766 replied:
Yes I think a channel for this initiative sounds great

neural-loop replied:
I created the channel, we can un-private it when the announcements go live.

neural-loop committed:
docs: Update review criteria checkboxes

neural-loop committed:
feat: Add Open Neuromorphic Research (ONR) program

Adds ONR program promotion and menu items.

Jegp commented on CONTRIBUTING-Staging.MD:
```diff
@@ -54,23 +53,30 @@ git push origin staging
 Pushing to the `staging` branch triggers the `Deploy Hugo site to Pages` GitHub Action. Once the action completes, your staging site will be live. You can find the URL in your fork's repository settings on the same **Settings > Pages** screen you configured in Step 1.
 
-## Handling the `PAT_FOR_ISSUES` Secret
+## Optional: Enabling the Mission Board on Your Staging Site
```
Nice!

Jegp commented on content/blog/announcing-the-onr-program/index.md:
```diff
@@ -0,0 +1,45 @@
+
+### How It Works
+
+The ONR program is built on a transparent review process managed through OpenReview. We welcome submissions of all kinds, from research papers and codebases to datasets and educational tools.
```
Maybe add a link to OpenReview? Something like
```
... managed through [OpenReview](https://openreview.net/), a review platform trusted by top-tier academic venues.
```

Jegp commented on content/neuromorphic-computing/research/guide/badge-system/index.md:
```diff
@@ -0,0 +1,42 @@
+
+## The Certificate Page
+
+Each project that successfully passes the ONR review is granted a permanent "certificate" page within our [Approved Research Registry](/neuromorphic-computing/research/papers/). This page serves as the official record of the community's endorsement and typically includes:
```
Should it be `/papers/`? What if people submit datasets/libraries/...? What about `/research/certificates`?

Feel free to ignore if this is nitpicking.

neural-loop replied:
It could have been certificates, I considered various options but felt that 'neuromorphic research papers' was a juicier SEO target. :)

Jegp commented on content/neuromorphic-computing/research/papers/example-paper-1/index.md:
```diff
@@ -0,0 +1,23 @@
+**Note:** This is a placeholder entry to demonstrate the layout and structure of the ONR Approved Research Registry.
+
+## Abstract
+The backpropagation of error algorithm is arguably the most important algorithm in artificial intelligence, but has been deemed biologically implausible. In this placeholder study, we introduce a novel, biologically plausible learning rule inspired by the dynamics of astrocytic networks. Our model, termed Astro-Modulated Hebbian Learning (AMHL), demonstrates competitive performance with backpropa ...
```
Lol!

Jegp reviewed (approved):
Minor comments on wording and links. Otherwise, LGTM!

DRosen766 committed:
update blog

DRosen766 committed:
add ONR to share your work

DRosen766 committed:
update dfp

neural-loop committed:
refactor: docs: Update ONR program and examples

This commit updates the ONR program announcement and example paper entries. It removes example papers 2 and 3 and modifies paper 1 to be an example of a certificate page. It also updates the ONR announcement post.

neural-loop committed:
Merge remote-tracking branch 'origin/main' into research-tmp

neural-loop committed:
docs: Update ONR program announcement

neural-loop committed:
docs

neural-loop committed:
docs: Update ONR program links and descriptions

neural-loop committed:
docs: Update ONR program announcement

neural-loop committed:
fix(badge-system): Corrected badge text

The badge text was updated from "ONM Community Approved" to "Open Neuromorphic Research".

neural-loop committed:
Refactor: Update badge and example certificate link

neural-loop committed:
feat(papers): Add noindex to example paper

neural-loop committed:
feat(blog): Add youtube video to ONR announcement

neural-loop committed:
feat: Add endorsements to ONR program

This commit adds an endorsements section to the ONR program page, including a new endorsements component and data file.

neural-loop committed:
feat: Implement endorsement slideshow and adjust styles

The endorsements section now features a slideshow for displaying multiple endorsements. The page header hero has been updated to hide the endorsements heading.

neural-loop committed:
docs: Update submitter guide with desk rejection note

neural-loop committed:
fix(blog): Update ONR link in announcement post

neural-loop committed:
fix: Update links in reviewer and submitter guides

neural-loop committed:
fix: Update endorsement from Eva to Jason

neural-loop committed:
fix: Correct review criteria links

neural-loop committed:
Merge branch 'main' into research-tmp

neural-loop committed:
fix: Update ONR announcement and endorsements

Body:
- Updated YouTube video ID in the ONR announcement blog post.
- Removed placeholder endorsements.

neural-loop committed:
fix: endorsements.toml: Update Jason Eshraghian's image path
